### PR TITLE
Create concise-guide-to-fixing-web-standards.md

### LINKED ISSUE #190: Concise Guide to Participating in Web Standards Document WIP
Taking inpsiration from the OpenSSF's [recent concise guides](https://github.com/ossf/wg-best-practices-os-developers/blob/main/docs/Concise-Guide-for-Evaluating-Open-Source-Software.md), we are collaborating on a "Concise Guide" of our own. 

Action: add suggested bullets for discussion / collaboration at next meeting [here](https://docs.google.com/document/d/1yDxUCnvZC23HfZXOz7OAbyWHZqqqNXBZHk0y3GBeIds/edit)

### PR DESCRIPTION
**Pull request by jorydotcom**
Creates directory for resources and adds concise guide per March 7 meeting 

Closes #190

### CODE REVIEW

File: resources/concise-guide-to-fixing-web-standards.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+If you’ve found a new bug - congratulations! Web platform bugs can be very fun problems to solve, but you may also find that you have to work a bit differently to resolve them. In addition to thoroughly understanding the bug, you may find that you have to do a lot of leg work to help others understand it - and agree to the resolution. It may require you to build consensus between engineers with competing implementations, contribute tests, or present research about usage patterns to spec proposal authors. 
+
+**Additional Resources:**
+
+
+
+* Web Platform Contribution Guide [https://wpc.guide/bug-guide/](https://wpc.guide/)
+* Web Compat Project [https://webcompat.com/](https://webcompat.com/)
+* How to Report a Bug [http://testthewebforward.org/docs/bugs.html](http://testthewebforward.org/docs/bugs.html)
+* Case Study: The Caper of the Flakey Test [https://bocoup.com/blog/the-caper-of-the-flaky-test](https://bocoup.com/blog/the-caper-of-the-flaky-test)
```
Comment (ljharb) [inline resources/concise-guide-to-fixing-web-standards.md:51]:
```suggestion
* Case Study: The Caper of the Flaky Test [https://bocoup.com/blog/the-caper-of-the-flaky-test](https://bocoup.com/blog/the-caper-of-the-flaky-test)
```

File: resources/concise-guide-to-fixing-web-standards.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+Copy over the current page, script, or even your entire project and start aggressively *removing* code, checking that you can still reproduce the bug with the code left. This also applies to the UI: Shorten any non-relevant text, eliminate any unrelated CSS styling, simplify any HTML subtree. You can almost always reduce more than you think. Most bugs can be reduced to &lt; 20 lines of code, often significantly less than that. If you’ve used unit tests before, it's a very similar process: imagine that you are writing a unit test to test if that particular bug is fixed.  
+
+
+
+    These additional resources may be helpful in learning how to create better reduced testcases: 
+	* [https://webkit.org/test-case-reduction/](https://webkit.org/test-case-reduction/)
+   * [https://css-tricks.com/reduced-test-cases/](https://css-tricks.com/reduced-test-cases/) 
+
+
+2. **What kind of bug do you have?** \
```
Comment (ljharb) [inline resources/concise-guide-to-fixing-web-standards.md:20]:
```suggestion
2. **What kind of bug do you have?**
```
i think all these backslashes at the end of lines need to be removed? (and trailing whitespace as a result)

File: resources/concise-guide-to-fixing-web-standards.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+ \
+Copy over the current page, script, or even your entire project and start aggressively *removing* code, checking that you can still reproduce the bug with the code left. This also applies to the UI: Shorten any non-relevant text, eliminate any unrelated CSS styling, simplify any HTML subtree. You can almost always reduce more than you think. Most bugs can be reduced to &lt; 20 lines of code, often significantly less than that. If you’ve used unit tests before, it's a very similar process: imagine that you are writing a unit test to test if that particular bug is fixed.  
+
+
+
+    These additional resources may be helpful in learning how to create better reduced testcases: 
+	* [https://webkit.org/test-case-reduction/](https://webkit.org/test-case-reduction/)
+   * [https://css-tricks.com/reduced-test-cases/](https://css-tricks.com/reduced-test-cases/)
```
Comment (ljharb) [inline resources/concise-guide-to-fixing-web-standards.md:17]:
```suggestion
	* [https://webkit.org/test-case-reduction/](https://webkit.org/test-case-reduction/)
    * [https://css-tricks.com/reduced-test-cases/](https://css-tricks.com/reduced-test-cases/) 
```

File: resources/concise-guide-to-fixing-web-standards.md (hunk 2 above)
Comment (jorydotcom) [reply resources/concise-guide-to-fixing-web-standards.md:20]: weird, that must've come from the docs-to-markdown generator tool....

File: resources/concise-guide-to-fixing-web-standards.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+5. **Is there a test for the feature you are using, and if so, is it passing or failing?** \
+Most modern web standards have either a test suite or a reference implementation associated with them: for HTML and CSS, the Web Platform Tests project contains the most authoritative test suite; Javascript’s test suite is called Test262. If there is a test suite, run tests relating to the bug you found in the implementation you were using.   Does your implementation run the tests correctly? If the implementation seems to be correct, double-check your application code - you may have found an issue with the way a library or tool you’re using has implemented the standard, rather than an issue with the standard itself.  
+
+
+    If there is a reference implementation, try the scenario in which you found a bug in the presumably standard-compliant reference implementation. If you see behavior you think is buggy in the reference implementation, the specification itself may need fixing. 
+
+
+6. **Have other developers reported a similar issue?** \
```
Comment (ljharb) [inline resources/concise-guide-to-fixing-web-standards.md:39]: there's still a bunch of spurious `\` in this doc

**Pull request merged**